packages/electron/app/package.json

```diff
@@ -8,7 +8,8 @@
   },
   "resolutions": {
     "electron": "3.1.0",
-    "debug": "^2.6.9"
+    "debug": "^2.6.9",
+    "xpc-connection": "sandeepmistry/node-xpc-connection#26/head"
```

Comment:
I have a vague, chilling memory that is telling me that this shouldn't be there. I think that will break the windows build.

Reply:
😂  - hmmm ok, whenever I `yarn` it adds it automatically.. but can remove from pr...

Reply:
@pazdera - so looking through the codebase it seems that that package is added here only if `platform === darwin` (macos?) - https://github.com/KanoComputing/kit-app-shell/blob/master/packages/electron/tasks/postinstall.js#L24 

But looks like its a optionalDep of `noble` for the electron package so everytime I yarn it gets added...

I guess what we could do is switch that script in postinstall to remove the dependancy from package.json if `platform === windows` if we do think it's a problem.

Right now though we're splitting the prod branches in `kit-app-ui` based on platform i.e `wand-windows-prod` / `wand-ios-prod` so I'll only bump the package for the ios branches - when you're in on weds lets discuss and see if this is something we need to retroactively address. For the moment though, just keen to get a fix out for ios14.